builtin/providers/rancher/resource_rancher_environment.go

```diff
@@ -85,6 +109,19 @@ func resourceRancherEnvironmentCreate(d *schema.ResourceData, meta interface{})
 	d.SetId(newEnv.Id)
 	log.Printf("[INFO] Environment ID: %s", d.Id())
 
+	// Add members
+	envClient, err := meta.(*Config).EnvironmentClient(d.Id())
+	if err != nil {
+		return err
+	}
+	members := d.Get("members").([]interface{})
```

Comment:
Okay, can we add both an acceptance test for the addition of multiple `member` blocks, as well as verification that the current acceptance tests still pass with this changeset?